packages/pv-scripts/helpers/prepareWebpackConfig.js

```diff
@@ -2,11 +2,34 @@ const webpackMerge = require("webpack-merge");
 const { getConfig, getCustomWebpackConfig } = require("@pro-vision/webpack-config");
 
 async function prepareWebpackConfig(mode) {
-  const customModeConfigName = mode === "development" ? "webpack.config.dev.js" : "webpack.config.prod.js";
+  const customModeConfigName = mode === "development" ? "webpack.config.dev" : "webpack.config.prod";
   const customWebpackConfig = await getCustomWebpackConfig("webpack.config.js");
-  const customWebpackDevConfig = await getCustomWebpackConfig(customModeConfigName);
+  const customWebpackModuleConfig = await getCustomWebpackConfig("webpack.config.module.js");
+  const customWebpackLegacyConfig = await getCustomWebpackConfig("webpack.config.legacy.js");
+  const customWebpackStageConfig = await getCustomWebpackConfig(`${customModeConfigName}.js`);
```

Comment:
wouldn't customWebpackModeConfig be more consistent? above it's also called customModeConfigName and not customStageConfigName

Reply:
fixed